lib/config/fileconf.go

```diff
@@ -1542,6 +1546,8 @@ type X11 struct {
 type Databases struct {
 	// Service contains common service fields.
 	Service `yaml:",inline"`
+	// ProxyTAG enables proxying of the TAG SQL interface.
+	ProxyTAG string `yaml:"proxy_tag,omitempty"`
```

Comment:
I followed existing patterns from our service config definitions here, I can make the comment more helpful if needed. We pretty regularly allow strings such as "yes" instead of booleans and use a function from `apiutils` to parse there, should we still do that?

Reply:
I'd rename the variable `ProxyTAGFlag` then, to match `EnabledFlag`.

Of note, however, is that `application_service.debug_app` is just declared as a bool, and we seem to be mostly fine with it.